.github/workflows/k8s-e2e.yml

```diff
@@ -143,7 +143,7 @@ jobs:
           docker exec kind-control-plane journalctl --no-pager -u containerd >> $log_dir/containerd.log
           docker exec kind-control-plane journalctl --no-pager -u kubelet >> $log_dir/kubelet.log
 
-          kubectl get pod test-pod -o yaml >> $log_dir/test-pod.log
+          kubectl get pod test-pod -o yaml >> $log_dir/test-pod.log || echo "test-pod may be deleted or not created"
```

Comment:
Should exit from the script once any command fails by `set -e` ?

Reply:
This step is for getting more logs, so it should continue even with errors.